src/Resource/NullResource.php

```diff
@@ -0,0 +1,14 @@
+<?php namespace League\Fractal\Resource;
```

Comment:
missing file header and namespace should be on its own line.

This package should have a `.php_cs` file and (preferably) enable StyleCI :)